ios/RCTConvert+BitmovinPlayer.swift

```diff
@@ -989,12 +1018,15 @@ extension RCTConvert {
      - Returns: The produced `SourceRemoteControlConfig` object
      */
     static func sourceRemoteControlConfig(_ json: Any?) -> SourceRemoteControlConfig? {
-        guard let json = json as? [String: Any?] else {
+        guard let json = json as? [String: Any?],
+              let castSourceConfigJson = json["castSourceConfig"] as? [String: Any?] else {
             return nil
         }
-
         return SourceRemoteControlConfig(
-            castSourceConfig: RCTConvert.sourceConfig(json["castSourceConfig"])
+            castSourceConfig: RCTConvert.sourceConfig(
+                json["castSourceConfig"],
+                drmConfig: RCTConvert.drmConfig(castSourceConfigJson["drmConfig"]).widevine
```

Comment:
Use only Widevine DRM config for casting as no other is supported for casting right now.